src/service/url-replacements-impl.js

```diff
@@ -806,6 +866,28 @@ export class GlobalVariableSource extends VariableSource {
     });
   }
 
+  /**
+   * Resolves the value via the user location service.
+   * @param {function(Object<string, string>)} getter
+   * @param {string} expr
+   * @return {!Promise<Object<string,(string|Array<string>)>>}
+   * @template T
+   * @private
+   */
+  getUserLocation_(getter, expr) {
+    const element = this.ampdoc.getHeadNode();
+    return Services.userLocationForDocOrNull(element).then(
+      userLocationService => {
+        userAssert(
+          location,
```

Comment:
Is this `window.location`? Might be more clear to be explicit here.

Reply:
oof no, it's supposed to be a local variable. I should probably change references from `location` to `position`